CMake/settings.cmake

```diff
@@ -1 +1,10 @@
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Werror -Wno-narrowing -Wno-dangling-else -Wno-deprecated-declarations")
+# set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Werror -Wno-narrowing -Wno-dangling-else -Wno-deprecated-declarations")
+
+target_compile_options(clio  
```

Comment:
This is a better way of setting flags for `clio` and derivatives (`clio_server`, `clio_tests`, etc.) only.

Reply:
This might work?
```
 PUBLIC -Wall
        -Werror 
        -Wno-narrowing
        -Wno-deprecated-declarations 
        -Wno-unused-local-typedefs 
        -Wno-unknown-pragmas 
        -Wno-dangling-else)
        ```